### DISCUSSION TITLE
A Code walkthrough for new contributors and people new to haskell

### DISCUSSION THREAD

**Discussion by webdeb** [hasura/graphql-engine#5848] [General] [answered]
This is just a request to the core team to provide a weekly dive-in video session about hasura, so people could learn about haskell and hopefully contribute to the project.

**Comment by abooij** [comment, marked as answer]
This is probably not sufficient, but previously I prepared a Codeflow that explains the GraphQL query execution model after our big [PDV refactor](https://github.com/hasura/graphql-engine/pull/4111). This document is still very relevant to development that is happening right now. Perhaps it helps a bit, so here it is:
https://beta.usecodeflow.com/lexi-lambda/graphql-engine/c/pdv-execution-model

Additionally, another resource is the server's [contributing guide](https://github.com/hasura/graphql-engine/blob/master/server/CONTRIBUTING.md), which will help you to get everything compiled and running in a development environment (not to be confused with the [general contributing guide](https://github.com/hasura/graphql-engine/blob/master/CONTRIBUTING.md)).

Of course, none of these resources come close in terms of helpfulness to a weekly dive-in session or in-depth documentation on the source code. I am just linking them for reference.
